fetch/fetchRequest.js

```diff
@@ -13,11 +13,18 @@
 var fetch = require('cross-fetch');
 
 function fetchRequest(method, url, args) {
+  var body = args.data;
+
+  // JSON encode body (if appropriate)
+  if (body && args.headers && args.headers['Content-Type'] === 'application/json' && typeof body !== 'string') {
```

Comment:
Are we sure it will be 'Content-Type' and never 'content-type'?  IIRC the spec is not case sensitive.

Reply:
Bump ^

Reply:
I can definitely make this change. But in reality we are the only users of this code. We use it as a pluggable utility wrapper around another xhr lib. Anything we do here, we must duplicate that logic in our jqueryRequest and reqwestRequest objects as well. Previously EVERYTHING was JSON encoded. I am creating a small pinhole for the token POST to succeed

Reply:
Shouldn't we check these values to be safe? If our dependency on cross-fetch, an application and/or plugin modifies the headers to be lowercase, this check will fail.